actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -176,6 +176,10 @@ def use_cookies_with_metadata
   #   Default is +false+.
   # * <tt>:httponly</tt> - Whether this cookie is accessible via scripting or
   #   only HTTP. Defaults to +false+.
+  # * <tt>:same_site</tt> - The value of the +SameSite+ cookie attribute, which
+  #   determines how this cookie should be restricted in cross-site contexts.
+  #   Possible values are +nil+, +:none+, +:lax+, and +:strict+. Defaults to
```

Comment:
FYI `nil` doesn't work. Proposing we use `false` instead in https://github.com/rails/rails/pull/45501